syntax = "proto3";
package to;

option java_package = "com.epocal.epoctest.to";
option java_outer_classname = "RealTimeQCTO";
option optimize_for = LITE_RUNTIME;

import "humidity_struct.proto";

message RealTimeQC {
    bool enabled = 1;
    int32 startTime = 2;
    int32 intervalTime = 3;
    int32 type = 4;                 // enum RealTimeQCType
    int32 numPoints = 5;
    double extra1 = 6;
    double extra2 = 7;
    double extra3 = 8;
    double extra4 = 9;
    double extra5 = 10;
    double extra6 = 11;
    bool continueIfFailed = 12;
    int32 humidityUntil = 13;
    repeated HumidityStruct humidityConfig = 14;
}
